CMakeLists.txt

```diff
@@ -47,6 +47,43 @@ if (${BUILD_VIEWER})
     COMMAND ${NPM} run build
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/view
   )
-endif ()
+endif () # ${BUILD_VIEWER}
+if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/view/build)
+  install(DIRECTORY view/build/ DESTINATION /var/lib/diffkemp/view
```

Comment:
Create a variable for `/var/lib/diffkemp/view` so that we can conveniently change it later if we decide to.